operators/community-windows-machine-config-operator/5.1.2/manifests/community-windows-machine-config-operator.5.1.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,349 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Seamless Upgrades
+    categories: OpenShift Optional
+    certified: "false"
+    createdAt: "2022-07-06T09:05:36Z"
+    description: An operator that enables Windows container workloads on OCP
+    olm.skipRange: '>=4.0.0 <5.1.1'
+    operatorframework.io/cluster-monitoring: "true"
+    operatorframework.io/suggested-namespace: openshift-windows-machine-config-operator
+    operators.operatorframework.io/builder: operator-sdk-v1.15.0+git
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    repository: https://github.com/openshift/windows-machine-config-operator
+    support: Red Hat
+  name: community-windows-machine-config-operator.v5.1.2
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions: {}
+  description: |-
+        ## WARNING
+        Community distribution of the Windows Machine Config Operator.
+        This is a preview build, and is not meant for production workloads.
+        Issues with this distribution of WMCO can be opened against the [WMCO repository](https://github.com/openshift/windows-machine-config-operator).
+        Please read through the [troubleshooting doc](https://github.com/openshift/windows-machine-config-operator/blob/community-4.10/docs/TROUBLESHOOTING.md)
+        before opening an issue.
+        Please ensure that when installing this operator the startingCSV you subscribe to is supported on the
+        version of OKD/OCP you are using. This CSV is meant for OKD/OCP 4.10.
+        ## Documentation
+        ### Introduction
+        The Windows Machine Config Operator configures Windows instances into nodes, enabling Windows container workloads
+        to be ran within OKD/OCP clusters. Windows instances can be added either by creating a [MachineSet](https://docs.openshift.com/container-platform/latest/machine_management/creating_machinesets/creating-machineset-aws.html#machine-api-overview_creating-machineset-aws),
+        or by specifying existing instances through a [ConfigMap](https://github.com/openshift/windows-machine-config-operator/blob/community-4.10/README.md#configuring-byoh-bring-your-own-host-windows-instances).
+        Through either method, the Windows instance must have the containerd container runtime installed. The operator will do
```

Comment:
This doc is not inline with what we have in the community-4.10 branch bundle csv [doc](https://github.com/openshift/windows-machine-config-operator/blob/79a9455d14ec3cc5f76ec4b8e8ceb2a8a47496a0/bundle/manifests/windows-machine-config-operator.clusterserviceversion.yaml#L27). Has this been manually updated?
We don't have a  pre-requisite for containerd runtime.

Reply:
The script I wrote uses this [description](https://github.com/openshift/windows-machine-config-operator/blob/master/hack/community/csv/description.md) in the generated csv. I can open a PR in our repo to have it use the description from the community-4.10 branch bundle csv doc that you linked. Thanks for pointing it out.

Reply:
On second thought, I think we should keep the description I had, but I will remove the containerd prerequisite. The one you linked tells users they need a Red Hat subscription. We should change that in the community-4.10 branch in our repo too.